### PR TITLE
Behonker and Clown Spider (ready for merge)

### PR DESCRIPTION
**Pull request by brainfood1183**
Adds two new mobs to the game, the Behonker which is a large floating honking death demon and the clown spider which is exactly as the name would suggest. The Behonker comes in 4 flavors and the spawner will spawn 1 at random.

Clown Spiders - clown spiders squeak when they move and can create clown spider webs which are pink slippery and edible. Clown spiders are now a rare occurance in ventcritters events.

Behonker - the behonker is a large beholder type demon of the honkmother. They can honk and fire laser beams from their eye. The behonker is unique in that it is also a living anomaly and when a behonker spawns it will spawn with 1 random set of anomaly traits. The behonker would make a good megafauna, admin event or expedition mob.

**Media**
untitled-235

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Clown Spiders have been sighted in the maintenance corridors.
- add: Rumors of giant floating Behonkers have been circulating between salvage crews.

### CODE REVIEW

**Review by Emisse — APPROVED** [review]

**Comment by deltanedas** [comment]
behonker elimination expedition when

File: Resources/Audio/Weapons/Guns/Gunshots/attributions.yml [hunk 1]
```diff
@@ -2,3 +2,8 @@
   license: "CC0-1.0"
   copyright: "Watering by elittle13. Converted to .OGG and MONO by EmoGarbage404 (github)"
   source: "https://freesound.org/people/elittle13/sounds/568558"
+
+- files: ["laser_clown.ogg"]
+  license: "CC0-1.0"
+  copyright: "made by brainfood1183 (github) for ss14"
```
Comment (metalgearsloth) [inline Resources/Audio/Weapons/Guns/Gunshots/attributions.yml:8]:
```suggestion
  copyright: "Made by brainfood1183 (github) for ss14"
```

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml [hunk 2]
```diff
@@ -1457,6 +1457,67 @@
...
+        Alive:
+          Base: clown
+        Critical:
+          Base: dead
+        Dead:
+          Base: dead
+    - type: MobThresholds
+      thresholds:
+        0: Alive
+        100: Critical
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/NPCs/animals.yml:1499]: Just remove crit state and have dead only.

File: Resources/Prototypes/Entities/Mobs/NPCs/behonker.yml [hunk 3]
```diff
@@ -0,0 +1,156 @@
+- type: entity
+  name: behonker
+  parent: SimpleSpaceMobBase
+  id: MobBehonkerBase
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/NPCs/behonker.yml:4]: BaseMobBehonker

File: Resources/Prototypes/Entities/Mobs/NPCs/behonker.yml [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+  abstract: true
+  description: A  floating demon aspect of the honkmother.
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/NPCs/behonker.yml:6]:
```suggestion
  description: A floating demon aspect of the honkmother.
```

File: Resources/Prototypes/Entities/Objects/Misc/spider_web.yml [hunk 5]
```diff
@@ -74,3 +74,96 @@
...
+          map: ["spiderWebLayer"]
+      drawdepth: WallMountedItems
+      netsync: false
+    - type: Appearance
+    - type: GenericVisualizer
+      visuals:
+        enum.SpiderWebVisuals.Variant:
+            spiderWebLayer:
+                1:  {state: spider_web_clown_1}
+                2:  {state: spider_web_clown_2}
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Misc/spider_web.yml:105]: plz don't do 4 space indentation, do 2.

File: Resources/Textures/Objects/Weapons/Guns/Battery/behonker_eye.rsi/meta.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    "copyright": "Made by brainfood1183 (github)",
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "states": [
+    {
+        "name": "base"
+    }
+    ]
```
Comment (metalgearsloth) [inline Resources/Textures/Objects/Weapons/Guns/Battery/behonker_eye.rsi/meta.json:13]: indentation

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Textures/Objects/Weapons/Guns/Battery/behonker_eye.rsi/meta.json (hunk 6 above)
Comment (brainfood1183) [reply Resources/Textures/Objects/Weapons/Guns/Battery/behonker_eye.rsi/meta.json]: fixed.

File: Resources/Prototypes/Entities/Objects/Misc/spider_web.yml (hunk 5 above)
Comment (brainfood1183) [reply Resources/Prototypes/Entities/Objects/Misc/spider_web.yml]: fixed.

File: Resources/Prototypes/Entities/Mobs/NPCs/behonker.yml (hunk 4 above)
Comment (brainfood1183) [reply Resources/Prototypes/Entities/Mobs/NPCs/behonker.yml]: fixed

File: Resources/Prototypes/Entities/Mobs/NPCs/behonker.yml (hunk 3 above)
Comment (brainfood1183) [reply Resources/Prototypes/Entities/Mobs/NPCs/behonker.yml]: fixed

File: Resources/Prototypes/Entities/Mobs/NPCs/animals.yml (hunk 2 above)
Comment (brainfood1183) [reply Resources/Prototypes/Entities/Mobs/NPCs/animals.yml]: removed crit state.

File: Resources/Audio/Weapons/Guns/Gunshots/attributions.yml (hunk 1 above)
Comment (brainfood1183) [reply Resources/Audio/Weapons/Guns/Gunshots/attributions.yml]: fixed.

**Review by metalgearsloth — APPROVED** [review]

**Comment by deltanedas** [comment]
@brainfood1183 any idea what to use for expedition structure objective?

**Comment by brainfood1183** [comment]
> @brainfood1183 any idea what to use for expedition structure objective?

You could use the bananium statues maybe, or i could make ya a new sprite?

**Comment by deltanedas** [comment]
bananium statue is fine no need for new sprite, might tweak it a little to be more cursed or smth

**Pull request merged**